portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -3018,6 +3018,32 @@ definition {
 		}
 	}
 
+	@description = "LPS-150510 Bug: Basic document cannot be selected in the subtype filter"
+	@priority = "4"
+	test FilterBySubtypeBasicDocument {
+		task ("Create a Basic Document") {
```

Comment:
As this is a priority 4 test, it should be included in the tango-acceptance routine. Please add:
 _property portal.acceptance = "true";_